src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -231,9 +236,15 @@ export class Table extends PureComponent<TableProps, TableState> {
           if (!canGoInApp) {
             return <EuiText size="s">{title || getDefaultTitle(object)}</EuiText>;
           }
-          return (
-            <EuiLink href={basePath.prepend(path)}>{title || getDefaultTitle(object)}</EuiLink>
-          );
+          let inAppUrl = basePath.prepend(path);
+          if (object.workspaces?.length) {
+            // first workspace login user have permission
+            const [workspaceId] = object.workspaces.filter((wsId) => visibleWsIds.includes(wsId));
```

Comment:
Do we need to consider use currentWorkspaceId here?

Reply:
you are right, add current workspace logic here